llvm/test/MC/AArch64/SVE2/directive-arch.s

```diff
@@ -19,3 +19,15 @@ rax1 z0.d, z0.d, z0.d
 .arch armv9-a+sve2-bitperm
 bgrp z21.s, z10.s, z21.s
 // CHECK: bgrp z21.s, z10.s, z21.s
+
+.arch armv9-a+sve2+f8f16mm
```

Comment:
I believe `+sve2` is a default of `armv9-a`, so it might not be necessary in these.